src/Orc.Controls/Controls/StepBar/Views/StepBar.xaml.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace Orc.Controls
+{
+    using System.Collections.Generic;
+    using System.Windows;
+    using System.Windows.Controls;
+
+    /// <summary>
+    /// Interaction logic for StepBar.xaml
+    /// </summary>
+    public sealed partial class StepBar
+    {
+        #region Constructors
+        public StepBar()
+        {
+            Loaded += OnLoaded;
+
+            InitializeComponent();
+        }
+        #endregion Constructors
+
+        #region Properties
+        public Orientation Orientation
+        {
+            get { return (Orientation)GetValue(OrientationProperty); }
+            set { SetValue(OrientationProperty, value); }
+        }
+
+        public static readonly DependencyProperty OrientationProperty = DependencyProperty.Register(nameof(Orientation), typeof(Orientation),
+            typeof(StepBar), new PropertyMetadata(Orientation.Vertical));
+
+        public IList<IStepBarItem> Items
+        {
+            get { return (IList<IStepBarItem>)GetValue(ItemsProperty); }
+            set { SetValue(ItemsProperty, value); }
+        }
+
+        public static readonly DependencyProperty ItemsProperty = DependencyProperty.Register(nameof(Items), typeof(IList<IStepBarItem>),
+            typeof(StepBar));
+
+        public bool AllowSelection
```

Comment:
I think this can be removed. AllowSelection will be determined on the Command on each IStepBarItem (and thus be available *per item*)